### DISCUSSION TITLE
How to output in textbox(winui 3) in a task?

### DISCUSSION THREAD

**Discussion by sxmike** [microsoft/WindowsAppSDK#2539] [Q&A]
I try to print some information in textbox in a task, using code like below:
 
DispatcherQueue.TryEnqueue(() => tbOutput.Text += Environment.NewLine + message);

But message does not show up immediately, instead all messages show up after the task completed.

How to do?

**Comment by castorix** [comment]
How do you launch the Task ?
I tested by setting it inside a [Periodic Timer ](https://docs.microsoft.com/en-us/windows/uwp/threading-async/create-a-periodic-work-item)and it seems to work fine, with this TextBox :

```
  <TextBox x:Name="tbOutput" Width="300" Height="200"  TextWrapping="Wrap" AcceptsReturn="True"
                      ScrollViewer.VerticalScrollMode="Enabled"
                    ScrollViewer.VerticalScrollBarVisibility="Auto" >
                
            </TextBox>
```
![PeriodicTimer_TextBox](https://user-images.githubusercontent.com/22345506/170318392-9af1e431-0270-4cf9-a819-ef77d1c56d16.gif)
